shell/packages/sandstorm-db/db.js

```diff
@@ -368,6 +368,14 @@ ApiTokens = new Mongo.Collection("apiTokens");
 //                  database when that happens. This field contains the boolean true to signify that
 //                  it has been set.
 //       ipInterface: Ditto IpNetwork, except it's an IpInterface.
+//       emailVerifier: An EmailVerifier capability that is implemented by the frontend. The
+//                      value is an object containing the field `services`, which itself is a
+//                      list of names of identity providers that are trusted to verify addresses.
+//                      If `services` is omitted or falsy, all configured identity providers are
```

Comment:
I might rename `services` to `restrictToServices` or `filterServices` to make it clearer in the consuming code that this is an optional filter, not an exhaustive list.  Not a blocker, though.